### DISCUSSION TITLE
Add Matic alongside XDai

### DISCUSSION THREAD

**Discussion by danimesq** [status-im/status-legacy#12632] [Ideas]

**Comment by cammellos** [comment]
thanks for the feature request, @hesterbruikman is that some token we would like to support?

**Comment by hesterbruikman** [comment]
Depends on what dapps have been deployed on Matic and whether they are commonly used among our user base. I suspect Decentraland is or will be available on Matic (don't know for sure), but that's a dapp that's not too convenient on mobile. Other than that, I don't know of dapps deployed to call for adding the network by default.

@DaniellMesquita do you know of other dapps deployed on Matic? Anything specific you're looking to use on the network?

@cammellos Regardless, we should check if manual entry works. Would be good if someone more familiar with adding networks can take a look. I'm running into the error: `Specified network id doesn't correspond to network id by RPC url` I have no clue what the network id should be. Tried 137 provided [here](https://docs.matic.network/docs/develop/network-details/network) as well as some other combinations. Always the same error.

![Screenshot_20201124-153558_Status](https://user-images.githubusercontent.com/35233054/100110046-ce743500-2e6c-11eb-8b28-e9373a0bc6d8.jpg)
